addons/main/functions/fnc_setUnconscious.sqf

```diff
@@ -41,6 +41,7 @@ if (_set) then {
                 };
             };
         }, [_unit, cba_missionTime], _restBleedout] call CBA_fnc_waitAndExecute;
+        _unit setVariable [QGVAR(bleedoutKillTime), cba_missionTime+_restBleedout];
```

Comment:
You want that to be network synced, that is the `true` at the end
```suggestion
        _unit setVariable [QGVAR(bleedoutKillTime), cba_missionTime + _restBleedout, true];
```

Also set the value to `nil` once they wake up, clean up and so on